bcos-pbft/bcos-pbft/pbft/cache/PBFTCache.cpp

```diff
@@ -302,8 +302,13 @@ void PBFTCache::resetCache(ViewType _curView)
 {
     m_submitted = false;
     m_precommitted = false;
-    if (!m_precommit && m_prePrepare && m_prePrepare->consensusProposal() &&
-        m_prePrepare->view() < _curView)
+    PBFT_LOG(INFO) << LOG_DESC("resetCache") << LOG_KV("precommit", m_precommit ? "true" : "false")
+                   << LOG_KV("prePrepare", m_prePrepare ? "true" : "false")
+                   << LOG_KV("curView", _curView);
+    if ((!m_precommit && m_prePrepare && m_prePrepare->consensusProposal() &&
```

Comment:
Please check this logic. I think it should add new logic:
auto validPreprepare = (m_precommit || (m_prePrepare && m_prePrepare->consensusProposal() && m_prePrepare->view() >= _curView));

if(validPreprepare && m_prePrepare->hash() == m_exceptionPrePrepare->hash())
{
   not reset the exceptionPreprepare
}
else
{
m_config->validator()->asyncResetTxsFlag(m_exceptionPrePrepare->consensusProposal()->data(), false);
}